rsts/howto/template_only_tasks.rst

```diff
@@ -0,0 +1,107 @@
+.. _howto-template-only-tasks:
```

Comment:
This does not belong to how to - but rather in - https://docs.flyte.org/projects/cookbook/en/latest/extending_flyte.html

Reply:
we should also update the decision flowchart to add this as a new type of extension possibility